doc/Table.md

```diff
@@ -0,0 +1,281 @@
+# The structure of database table
+
+## boinc_vars
+| Field                   | Type    | Description
+|-------------------------|---------|-----------
+| workunitName            | text    | The work unit name 
+| fpopsEstimate           | double  | Getting the fpops estimate and multiplying it with 10 to get the bound 
+| fpopsBound              | double  | fpopsEstimate*1000 
+| memBound                | int(11) | The workunit will only be sent to clients with at least this much available RAM. If exceeded the workunit will be aborted 
+| diskBound               | int(11) | An upper bound on the amount of disk space requred to process the workunit 
+| delayBound              | int(11) | An upper bound on the time (in seconds) between sending a result to a client and receiving a reply 
+| redundancy              | int(11) | The number of redundant calculations. Set to two or more ot achieve redundancy 
+| copies                  | int(11) | The number of copies of the workunit to issue to clients
+| errors                  | int(11) | The number of errors from clients before the workunit is declared to have an error 
+| numIssues               | int(11) | The total number of clients to issue the work unit to before it is declared to have an error 
+| resultsWithoutConcensus | int(11) | The total number of returned results without a concensus is found before the workunit is declared to have an error 
+| appName                 | text    | The app name (usually is sixtrack) 
+| appVer                  | int(11) | The app version 
+
+## env
+| Field          | Type    | Description
+|----------------|---------|------------
+| madx_exe       | text    | The madx executable
+| sixtrack_exe   | text    | The sixtrack executable 
+| study_path     | text    | The absolute path of this study 
+| preprocess_in  | text    | The input folder for preprocess jobs 
+| preprocess_out | text    | The output folder for preprocess jobs 
+| sixtrack_in    | text    | The input folder for sixtrack jobs 
+| sixtrack_out   | text    | The output folder for sixtrack jobs 
+| gather         | text    | The input folder for gather job which is used for collecting results 
+| templates      | text    | The template folder 
+| boinc_spool    | text    | The boinc spool for submitting jobs to boinc 
+| test_turn      | int(11) | When submitting jobs to boinc, we need to test the jobs on HTCondor firstly 
+| emit           | double  | Normalized emittance 
+| gamma          | double  | The gamma 
+| kmax           | int(11) | The number of phase space angles 
+| boinc_work     | text    | The absolute path of work directory 
```

Comment:
maybe for `boinc_work` and `boinc_results`, please state that these are the spool directories on the work.boinc volume